arch/arm/src/cxd56xx/cxd56_gnss.c

```diff
@@ -2787,8 +2784,8 @@ static ssize_t cxd56_gnss_read(struct file *filep, char *buffer,
 
   ret = fw_gd_readbuffer(type, offset, buffer, len);
 
-  _err:
-  _success:
+err:
+success:
```

Comment:
Maybe we can use a single `out` label here?

Reply:
Done.